src/classes/RD2_RecurringDonations_TDTM.cls

```diff
@@ -102,4 +104,21 @@ public class RD2_RecurringDonations_TDTM extends TDTM_Runnable {
         }
     }
 
+    /***
+    * @description Creates schedule record after insert of recurring donation
+    * @param rds Trigger.new records
+    * @param triggerAction BeforeInsert, AfterInsert, etc.
+    */
+    private void createRecurringDonationSchedule(List<SObject> rds, TDTM_Runnable.Action triggerAction) {
```

Comment:
You can specify the parameter as `List<npe03__Recurring_Donation__c>` to avoid the cast in the for loop